docs/blog/2019-06-25-profitable-agency-wordpress/index.md

```diff
@@ -9,7 +9,7 @@ tags:
   - aws
 ---
 
-import { Link } from "gatsby"
+import { Link as GatsbyLink } from "gatsby"
```

Comment:
why was this changed to GatsbyLink?

Reply:
I changed this to get around the issue https://github.com/prettier/prettier/issues/7548

<Link> </Link> was being treated as <link /> (head tag) which cannot have children.

Reply:
Can we add prettier ignore instead? I feel we'll lose context on why this is here perhaps add a comment to the prettier issue.

Reply:
I can add a comment above this. But do we really need that context?
Using `Link` from gatsby like this is still correct and we wouldn't really need to do anything after we bump prettier with a version with a fix for this. Adding a comment or `prettier-ignore` would require an additional step after the `prettier` bump.

Just a thought. Would still happily update the PR. What is your opinion?

Reply:
> I can add a comment above this. But do we really need that context?

I think we should, workarounds should always tell us why this is needed (a comment is fine). It seems a bit overkill for this one because it's already fixed but it's all about consistency. I can imagine someone stumble on this line of code and having to think about why we did `import { Link as GatsbyLink } from "gatsby"` and not `import { Link } from "gatsby"`

Reply:
Okay. Updated.